config/locales/en.yml

```diff
@@ -19,6 +19,13 @@ en:
       not_confirmed: "A confirmation email was sent to your account at '%{email}'. You must follow the instructions in the email before your account can be activated"
     confirmations:
       invalid_token: "Invalid confirmation token. Please try again"
+      user_not_found: "Unable to find user with email '%{email}'."
+      sended: "An email has been sent to '%{email}' containing instructions for confirming your account."
```

Comment:
THis one is not used and should be removed, but should be `sent` instead of `sended`

Reply:
I agree that it should be sent. "sended" was originally referenced somewhere and wasn't sure if I should change it to "sent" or not.

Reply:
Removed this and rewrote the code to fall back on Devise's `confirmations.send_instructions` since it already exists there.